files/blink.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The desktop file & metainfo should go upstream if possible.

Reply:
Sorry for the ignorance, but what do you mean? I'm not familiar with this part of git..

Reply:
You should try and contribute these to the host project as a pull request or bug fix so they're included automatically in future versions. It's not just flatpak that uses these files, they improve things for deb packages, rpms, snaps, all sorts of things.